<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="CNameEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CSurnameEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CIDEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CIDLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>60</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="CNameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>76</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="CSurnameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>91</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Surname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertCustomerButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Customer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertRoomButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Room</string>
    </property>
   </widget>
   <widget class="QLabel" name="RIDLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>41</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Room ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="RNumLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>67</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Room Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="RFlLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>54</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Room Floor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RIDEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RNumEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RFlEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RCapLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>72</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Room Capacity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RCapEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ResRIDEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>277</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ResCIDEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>277</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="DateFrom">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>277</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="DateTo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>277</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>257</y>
      <width>50</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Date From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>257</y>
      <width>37</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Date To</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>257</y>
      <width>40</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Room Id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>59</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>177</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Reservation : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertReservation">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Reservation</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
